<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyrosQT</class>
 <widget class="QMainWindow" name="PyrosQT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyrosQT</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionOpen_Database"/>
    <addaction name="actionNew_Database"/>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Preferences</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionNew_Search"/>
   </widget>
   <widget class="QMenu" name="menuTag">
    <property name="title">
     <string>Tag</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionImport_Files"/>
    <addaction name="actionFind_Duplicates"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuDatabase"/>
   <addaction name="menuPreferences"/>
   <addaction name="menuView"/>
   <addaction name="menuTag"/>
   <addaction name="menuFiles"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionOpen_Database">
   <property name="text">
    <string>Open Database</string>
   </property>
  </action>
  <action name="actionNew_Database">
   <property name="text">
    <string>New Database</string>
   </property>
  </action>
  <action name="actionNew_Search">
   <property name="text">
    <string>New Search</string>
   </property>
  </action>
  <action name="actionImport_Files">
   <property name="text">
    <string>Import Files</string>
   </property>
  </action>
  <action name="actionFind_Duplicates">
   <property name="text">
    <string>Find Duplicates</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
